.travis.yml

```diff
@@ -4,12 +4,12 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-27.0.3
-    - android-27
+    - build-tools-29.0.2
+    - android-30
```

Comment:
`android-30` with `build-tools-29.*`? Are there `build-tools-30.*` or no?

Reply:
Also, should we use `BUILD_TOOLS_VERSION` here?

Reply:
Lol ignore me, this is in the travis file

Reply:
build-tools-30 is in RC so i did not wanna use that yet. But can upgrade to that later. :)